### PR TITLE
Adding unnecessary files

### PR DESCRIPTION
**Pull request by PublisherApplications**
### Description
I got bored and had it on necessary files...
 
### Issues Resolved
no issues
 
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass
- [x] New functionality has been documented.
  - [x] New functionality has javadoc added
- [x] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/search-processor/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: .github/draft-release-notes-config.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# The overall template of the release notes
+# The overall template of the release notes.
 template: |
   Compatible with OpenSearch (**set version here**).
   $CHANGES
 
-# Setting the formatting and sorting for the release notes body
+# Setting the formatting and sorting for the release notes body.
```
Comment (PublisherApplications) [inline .github/draft-release-notes-config.yml:6]: Adding punctuation...

File: .github/draft-release-notes-config.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ replacers:
   - search: '##'
     replace: '###'
 
-# Organizing the tagged PRs into categories
+# Organizing the tagged PRs into categories.
```
Comment (PublisherApplications) [inline .github/draft-release-notes-config.yml:15]: Adding punctuation..

File: .github/draft-release-notes-config.yml [hunk 3]
```diff
@@ -40,4 +40,4 @@ categories:
       - 'maintenance'
   - title: 'Refactoring'
     labels:
-      - 'refactoring'
\ No newline at end of file
+      - 'refactoring'
```
Comment (PublisherApplications) [inline .github/draft-release-notes-config.yml:43]: adding random things...

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,2 +1,2 @@
 # This should match the MAINTAINERS.md file in the root of this repo
-*   @msfroh @macohen @noCharger @mingshl @sejli
\ No newline at end of file
+*   @msfroh @macohen @noCharger @mingshl @sejli
```
Comment (PublisherApplications) [inline .github/CODEOWNERS:2]: Adding punctuation

**Review by PublisherApplications — COMMENTED** [review]
This is context about my changes.

**Comment by msfroh** [comment]
Spambot

**Pull request closed without merging**